app/helpers/hyrax/work_form_helper.rb

```diff
@@ -99,110 +99,20 @@ def form_file_set_select_for(parent:)
     end
 
     ##
-    # This helper retrieves errors based on form type.
-    #
-    # @param form [Hyrax::Forms::WorkForm] or Hyrax::ChangeSet
-    # @return [String] specific error message
-    def in_works_ids_errors_for(form:)
-      case form
-      when Hyrax::ChangeSet
-        form.errors[:in_works_ids]
-      else
-        form.model.errors[:in_works_ids]
-      end
-    end
-
-    ##
-    # This helper retrieves errors based on form type.
-    #
-    # @param form [Hyrax::Forms::WorkForm] or Hyrax::ChangeSet
-    # @return [String] specific error message
-    def ordered_member_ids_errors_for(form:)
-      case form
-      when Hyrax::ChangeSet
-        form.errors[:ordered_member_ids]
-      else
-        form.model.errors[:ordered_member_ids]
-      end
-    end
-
-    ##
-    # This helper retrieves errors based on form type.
-    #
-    # @param form [Hyrax::Forms::WorkForm] or Hyrax::ChangeSet
-    # @return [String] specific error message
-    def visibility_errors_for(form:)
-      case form
-      when Hyrax::ChangeSet
-        form.errors[:visibility]
-      else
-        form.model.errors[:visibility]
-      end
-    end
-
-    ##
-    # This helper retrieves errors based on form type.
-    #
-    # @param form [Hyrax::Forms::WorkForm] or Hyrax::ChangeSet
-    # @return [String] specific error message
-    def embargo_release_date_errors_for(form:)
-      case form
-      when Hyrax::ChangeSet
-        form.errors[:embargo_release_date]
-      else
-        form.model.errors[:embargo_release_date]
-      end
-    end
-
-    ##
-    # This helper retrieves errors based on form type.
-    #
-    # @param form [Hyrax::Forms::WorkForm] or Hyrax::ChangeSet
-    # @return [String] specific error message
-    def visibility_after_embargo_errors_for(form:)
-      case form
-      when Hyrax::ChangeSet
-        form.errors[:visibility_after_embargo]
-      else
-        form.model.errors[:visibility_after_embargo]
-      end
-    end
-
-    ##
-    # This helper retrieves errors based on form type.
-    #
-    # @param form [Hyrax::Forms::WorkForm] or Hyrax::ChangeSet
-    # @return [String] specific error message
-    def lease_expiration_date_errors_for(form:)
-      case form
-      when Hyrax::ChangeSet
-        form.errors[:lease_expiration_date_errors]
-      else
-        form.model.errors[:lease_expiration_date_errors]
-      end
+    # @param form [Hyrax::Forms::WorkForm]
+    # @return [String] specific error message(s)
+    def form_errors(form:)
+      error_messages = []
+      form.errors.messages.each_value { |v| error_messages << v.first }
```

Comment:
whether the form is a `Hyrax::Forms::WorkForm` or `Hyrax::ChangeSet`, `form.errors.messages` is the way to find the error messages. see my test below.

<details>
<summary> koppie </summary>

![image](https://github.com/samvera/hyrax/assets/29032869/e5bf6d8d-02db-4890-9986-99173dad3d7c)
</details>

<details>
<summary> hyrax </summary>

![image](https://github.com/samvera/hyrax/assets/29032869/ff47a9e8-b868-45a3-9fb1-0a8c51f33d79)
</details>